package.json

```diff
@@ -1,16 +1,15 @@
 {
   "name": "hof-form-wizard",
-  "version": "4.1.0",
+  "version": "4.2.0",
   "description": "routing and request handling for a multi-step form processes",
   "main": "index.js",
   "scripts": {
-    "test": "npm run lint && npm run cover && npm run check-coverage && npm run test:functional",
-    "test:unit": "_mocha",
-    "lint": "eslint .",
-    "cover": "istanbul cover _mocha -- --recursive --require test/helpers",
+    "test": "npm run test:lint && npm run test:unit && npm run check-coverage && npm run test:functional",
+    "test:lint": "eslint .",
+    "test:unit": "istanbul cover _mocha -- --recursive --require test/helpers",
     "check-coverage": "istanbul check-coverage --statement 85.53 --branch 81.58 --function 66.67 --line 86.21",
```

Comment:
And this one.

Reply:
yeah I was going to do that but I wasn't sure what the hell it did

Reply:
this is all it does when I run it 
```
> hof-form-wizard@4.2.0 check-coverage /Users/Sulthan/Development/projects/HO/hof-form-wizard
> istanbul check-coverage --statement 85.53 --branch 81.58 --function 66.67 --line 86.21
```

Reply:
It checks the coverage.

Specifically it checks that test coverage exceeds the values passed as flags. The clue that it's test related - and so should be prefixed - is that it's part of the `npm test` command.

Reply:
The output is minimal when it's passing - much like eslint in that regard.

Reply:
ahh Joe explained it, the numbers in the flags are a percentage.  So the test have to cover coverage of at least 85.53% etc, cool.

Good point about looking at the npm test command.  That now makes sense